components/TabBar/constants.js

```diff
@@ -41,3 +41,6 @@ export const TAB_BAR_BROWSE_ICON = `${TAB_BAR}.${BROWSE_ICON}`;
 export const TAB_BAR_CART_ICON = `${TAB_BAR}.${CART_ICON}`;
 export const TAB_BAR_MORE_ICON = `${TAB_BAR}.${MORE_ICON}`;
 export const TAB_BAR_FAVORITES_ICON = `${TAB_BAR}.${FAVORITES_ICON}`;
+
+export const TAB_BAR_SHOW_ACTION = 'TAB_BAR_SHOW_ACTION';
```

Comment:
The common pattern for actions would be `SHOW_TAB_BAR` and `HIDE_TAB_BAR`. Even if the rest of the constants here follow another pattern, i'd prefer the default one - without the `_ACTION` suffix  (like you also did for the `AddToCartBar`)

Reply:
For the coverage comment: I won't, I'd have to do some major fixing, all the related tests are already skipped for some reason. I'm afraid this would take too much time.